src/lib/mealy-fsm/mealy-output.fn.ts

```diff
@@ -0,0 +1,17 @@
+import { MachineEvent, MachineOutputFn, MachineState } from '../state-machine-api';
+import { MealyTransitionMap } from './mealy-transition-map';
+
+export const createMealyOutput: <T>(transitions: MealyTransitionMap<T>) => MachineOutputFn<T> = <T>(
+  transitions: MealyTransitionMap<T>,
+) => (currentState: MachineState<T>, $event: MachineEvent): T => {
+  if (!transitions.has(currentState.currentNode)) {
+    return Object.create(null);
```

Comment:
should this fail ? uhm.. so this happens when no output is defined for a transition. No idea if this should be allowed or just null is fine.

Reply:
Not sure anyway if we will access that map node, it is something that we have to check.